src/main/java/com/dreammaster/scripts/ScriptIndustrialCraft.java

```diff
@@ -1330,17 +1331,6 @@ public void loadRecipes() {
                 'h',
                 getModItem(GregTech.ID, "gt.metaitem.01", 1, 32602, missing));
 
-        GT_Values.RA.addFuel(
-                createItemStack(
-                        IndustrialCraft2.ID,
-                        "itemFluidCell",
-                        1,
-                        0,
-                        "{Fluid:{FluidName:\"ic2biogas\",Amount:1000}}",
-                        missing),
-                getModItem(IndustrialCraft2.ID, "itemFluidCell", 1, 0, missing),
-                40,
-                1);
```

Comment:
Is removal of biogas as fuel intended?

Reply:
`addFuel` with `aType` 1 means adding to `sTurbineFuels`. And we already have biogas turbine fuel recipe, added by GT.